src/learn/qml/MuseScore/Learn/internal/ClassesPage.qml

```diff
@@ -63,141 +64,159 @@ FocusScope {
         }
     }
 
-    Rectangle {
-        anchors.top: parent.top
-        anchors.left: parent.left
-        anchors.leftMargin: root.sideMargin
+    ScrollBar.vertical: StyledScrollBar {
+        parent: root.parent
+
+        anchors.top: root.top
+        anchors.bottom: root.bottom
         anchors.right: parent.right
-        anchors.rightMargin: root.sideMargin
 
-        height: authorInfo.height
+        visible: root.contentHeight > root.height
+        z: 1
+    }
 
-        radius: 12
+    header: Item {
+        height: authorInfoContainer.height
+        anchors.left: parent.left
+        anchors.right: parent.right
+        Rectangle {
+            id: authorInfoContainer
+            anchors.top: parent.top
+            anchors.left: parent.left
+            anchors.leftMargin: root.sideMargin
+            anchors.rightMargin: root.sideMargin
 
-        color: ui.theme.backgroundPrimaryColor
+            height: authorInfo.height
+            width: (parent.width > 1100) ? 1000 : parent.width - 96
 
-        Column {
-            id: authorInfo
+            radius: 12
 
-            anchors.left: parent.left
-            anchors.leftMargin: 36
-            anchors.right: parent.horizontalCenter
-            anchors.top: parent.top
-            anchors.topMargin: 36
+            color: ui.theme.backgroundPrimaryColor
 
-            height: childrenRect.height + 72
+            Column {
+                id: authorInfo
 
-            spacing: 30
+                anchors.left: parent.left
+                anchors.leftMargin: 36
+                anchors.rightMargin: 36
+                anchors.right: parent.right
```

Comment:
This is a real nitpick, but if you're making amendments anyway, could you then please swap these two lines to make the order of the properties consistent for all three anchors? 
```suggestion
                anchors.right: parent.right
                anchors.rightMargin: 36
```

Reply:
I completely feel you here.